account_payment_return/README.rst

```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============================
+Customer Payment Orders Returns
+===============================
+
+This module implements the return of customer receivables and allows to catch
+several reconciled account move lines, to give back to a state where the debt
+is still open, and letting history of it.
+
+This module can be extended adding importers that fills automatically the
+full payment return record.
```

Comment:
not sure about "return" wording.
Payback? reimbursement?
Anglophone speaker wanted ;)